app/validators/api/v2025_0_rc/hesa_trainee_detail_attributes/rules/funding_method.rb

```diff
@@ -63,6 +63,25 @@ def no_funding_method?
           def funding_type
             @funding_type ||= FUNDING_TYPES[funding_method]
           end
+
+          def academic_cycle
+            @academic_cycle ||= start_date.present? ? AcademicCycle.for_date(start_date) : AcademicCycle.for_date(Time.zone.now + ::Trainees::SetAcademicCycles::DEFAULT_CYCLE_OFFSET)
+          end
+
+          def start_date
+            value = trainee_attributes.trainee_start_date || trainee_attributes.itt_start_date
+            if value.is_a?(String)
+              value = valid_date_string?(value) ? Date.parse(value) : nil
```

Comment:
```suggestion
              value = valid_date_string?(value) ? Date.8601(value) : nil
```

Reply:
Assuming you mean this 😄 
```suggestion
              value = valid_date_string?(value) ? Date.iso8601(value) : nil
```